e2e/init.js

```diff
@@ -0,0 +1,13 @@
+import detox from "detox";
+import { detox as config } from "../package.json";
+
+// Set the default test timeout of 120s
+jest.setTimeout(120000);
```

Comment:
out of curiosity, why 120 seconds ?

Reply:
Abitrary, e2e timeouts are usually quite high to allow for network latency.